frontend/src/routes/Profile.jsx

```diff
@@ -0,0 +1,437 @@
+import { useState, useEffect } from 'react';
+import '../styles/ProfilePage.css';
+import Header from '../components/Header';
+import { serverFetch } from '../hooks/serverUtils';
+import { getHash } from '../hooks/serverUtils';
+import PropTypes from 'prop-types';
+
+function Profile() {
+  return (
+    <>
+      <UserForm></UserForm>
+    </>
+  );
+}
+
+function UserForm() {
+  //Route needs to be modified when it's implemented od backend
```

Comment:
Please add whitespace after `//` -> `// Route needs to ...`

The linter/formatter should do this automatically but I guess not -.-